File: Kattis/CPP/IntroProblems/oddities.cc
/*
kattis oddities problem
Author: Agis Daniels
Solve read in n lines and test if each is odd or even
NOTE 
*/

#include <bits/stdc++.h>

using namespace std;

int main(){
    int n,x;
    cin>>n;
    while(n--){
        cin>>x;
        string ans=(x&1)? "odd": "even";
        cout<<x<<" is "<<ans<<endl;
    }
    return 0;
}

File: Kattis/Python/IntroProblems/oddities.py
""" 
kattis oddities problem
Author: Agis Daniels
Solve read in n lines and test if each is odd or even
NOTE 
"""

from sys import stdin as rf
def main():
    mp={True:'even',False:'odd'}
    n=int(rf.readline())
    for line in rf:
        x=int(line)
        print("{} is {}".format(x, mp[x%2==0]))
if __name__=='__main__':
    main()
